packages/ckeditor5-language/src/language.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * @license Copyright (c) 2003-2021, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module language/language
+ */
+
+import { Plugin } from 'ckeditor5/src/core';
+
+import LanguageEditing from './languageediting';
+import LanguageUI from './languageui';
+
+/**
+ * The language plugin.
+ *
+ * For more information about this feature check the {@glink api/language package page}.
+ *
+ * This is a "glue" plugin which loads the {@link module:language/languageediting~LanguageEditing language editing feature}
+ * and {@link module:language/languageui~LanguageUI language UI feature}.
+ *
+ * @extends module:core/plugin~Plugin
+ */
+export default class Language extends Plugin {
+	/**
+	 * @inheritDoc
+	 */
+	static get requires() {
+		return [ LanguageEditing, LanguageUI ];
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	static get pluginName() {
+		return 'Language';
+	}
+}
+
+/**
+ * The configuration of the language feature. Introduced by the {@link module:language/languageediting~LanguageEditing} feature.
+ *
+ * Read more in {@link module:language/language~LanguageConfig}.
+ *
+ * @member {module:language/language~LanguageConfig} module:core/editor/editorconfig~EditorConfig#language
+ */
+
+/**
+ * The configuration of the language feature.
+ * The option is used by the {@link module:language/languageediting~LanguageEditing} feature.
+ *
+ *		ClassicEditor
+ *			.create( {
+ * 				language: ... // Language feature config.
+ *			} )
+ *			.then( ... )
+ *			.catch( ... );
+ *
+ * See {@link module:core/editor/editorconfig~EditorConfig all editor options}.
+ *
+ * @interface LanguageConfig
+ */
+
+/**
+ * The available language options.
+ *
+ * The default value is:
+ *
+ *		const languageConfig = {
+ *			options: [
+ *				{ title: 'Arabic', class: 'ck-language_ar', languageCode: 'ar' },
```

Comment:
Where is that class used? I'm not sure the `ck-` prefix makes sense, but it depends on the context.

Reply:
I should've read further:

```
 * The `title` and `class` properties will be used by the `language` dropdown to render available options.
```

I don't think anyone will be styling these options in the dropdown so I don't think we need to have specific classes for them. If anything, we can have these classes added automatically.

In the heading feature, the classes are configurable as you're supposed to style the headings in the dropdown to look like headings in the content. And for that you need these classes.